Library/Formula/orfeo.rb

```diff
@@ -1,35 +1,25 @@
 class Orfeo < Formula
   desc "Library of image processing algorithms"
   homepage "http://www.orfeo-toolbox.org/otb/"
-  url "https://downloads.sourceforge.net/project/orfeo-toolbox/OTB/OTB-3.20/OTB-3.20.0.tgz"
-  sha256 "7c405756887842fbdfd6eb99a1fdc3c940817290a99c188414e5af45702d1b9d"
+  url "https://downloads.sourceforge.net/project/orfeo-toolbox/OTB/OTB-5.0.0/OTB-5.0.0.tgz"
+  sha256 "35e2f1d280f4a33b3d38af840a83069b34b2ea99acba18b1a944acccdbf1a976"
+
+  option "with-examples", "Compile and install various examples"
```

Comment:
You'll need to deprecate the old options onto the new options here with the:

```ruby
deprecated_option "examples" => "with-examples"
```
style syntax.